source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -1921,6 +1922,27 @@ std::function<std::unique_ptr<OnProcessingResponse>()> FilterConfig::createOnPro
   };
 }
 
+std::unique_ptr<AttributeBuilder> FilterConfig::createAttributeBuilder(
+    const ExternalProcessor& config,
+    Extensions::Filters::Common::Expr::BuilderInstanceSharedConstPtr builder,
+    Server::Configuration::CommonFactoryContext& context) {
+  if (config.has_attribute_builder()) {
+    auto& factory = Envoy::Config::Utility::getAndCheckFactory<AttributeBuilderFactory>(
```

Comment:
your call: we could also create the default builder using a default factory, then we could remove the down-below else branch.